src/pages/reference-guides/ReferenceGuideMenu.tsx

```diff
@@ -71,7 +71,7 @@ export default class ReferenceGuideMenu extends I18nMixin(WidgetBase)<ReferenceG
 
 		return (
 			<SideMenu>
-				<SideMenuSection>{list}</SideMenuSection>
+				<SideMenuSection fixed={true}>{list}</SideMenuSection>
```

Comment:
Nit: This should be changed to `fixed` or other places should be changed to `fixed={true}` for consistency (I prefer the former.)